src/main/java/Main.java

```diff
@@ -0,0 +1,34 @@
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+import krona.Krona;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private Krona krona = new Krona("data/tasks.txt");
```

Comment:
I'm not sure if its my side, but when I used the Bye Command with your Krono, it was unable to save the tasks into a task.txt, and I couldn't find a tasks.txt in the directory, so maybe you can add it to your git? Unless I'm blind in which I sincerely apologise!

Reply:
Hey! Am i supposed to add the data/tasks.txt to git? i just checked and it was under ignored files because i assume its supposed to be for each individual computer